internal-payment-sdk/sdk/src/main/java/net/gini/android/internal/payment/GiniInternalPaymentModule.kt

```diff
@@ -200,20 +233,73 @@ class GiniInternalPaymentModule(private val context: Context,
                     it
                 )
             } ?: IngredientBrandType.INVISIBLE
+
+        fun saveSDKCommunicationTone(tone: String) {
+            sharedPreferences.edit().apply {
+                putString(SDK_COMMUNICATION_TONE_PREFS_KEY, tone)
+                apply()
+            }
+        }
+
+        fun getSDKCommunicationTone(): CommunicationTone =
+            sharedPreferences.getString(
+                SDK_COMMUNICATION_TONE_PREFS_KEY,
+                CommunicationTone.FORMAL.toString()
+            )?.let {
+                CommunicationTone.valueOf(
```

Comment:
[Request] Similar to the crash described [here](https://github.com/gini/gini-mobile-android/pull/646#pullrequestreview-2546441940) , the app will crash if backend sends an invalid value for `CommunicationTone`. 
Please consider handling this case, and returning a default value.